_posts/2023-12-22-central-workflows.markdown

```diff
@@ -0,0 +1,197 @@
+---
+layout: post
+title:  "DRY in GitHub Actions with reusable workflows"
+date:   2023-12-22
```

Comment:
Would you mind moving this to January? I think we'd be able to create a bit more buzz and get more traffic on this article, if we'd post it early Jan, once people are back from their holidays